browser/resources/settings/email_aliases_page/content/email_aliases_modal.tsx

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import Button from "@brave/leo/react/button"
+import Icon from "@brave/leo/react/icon"
+import { getLocale } from '$web-common/locale'
+import formatMessage from '$web-common/formatMessage'
+import * as React from 'react'
+import { ViewState, ViewMode, MappingService, MAX_ALIASES } from "./types"
+import Col from "./styles/Col"
+import Row from "./styles/Row"
+import Input from "@brave/leo/react/input"
+import styled from "styled-components"
+import onEnterKey from "./onEnterKey"
+import { color, font, spacing } from "@brave/leo/tokens/css/variables"
+
+
+const ModalSectionCol = styled(Col)`
+  margin: ${spacing["2Xl"]} 0;
+  & h3 {
```

Comment:
I don't think you can trust this has the right font settings, so probably worth updating it with what's in the design

Reply:
removed (font settings fixed elsewhere)